### PR TITLE
Create identifier during build for da00 flatbuffer

### PR DESCRIPTION
**Pull request by tbukovics**
### Issue reference / description

This pull request corrects a bug of missing identifier.

## Checklist for submitter

- [ ] Check for conflict with integration test
- [ ] Unit tests pass

---

### Nominate for Group Code Review

- [ ] Nominate for code review

### CODE REVIEW

File: src/common/kafka/serializer/DA00HistogramSerializer.h [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 #pragma once
 
+#include "da00_dataarray_generated.h"
```
Comment (mortenjc) [inline src/common/kafka/serializer/DA00HistogramSerializer.h:17]: <> ?
- I know it is not super consistent for the _generated.h files, but ...

File: src/common/kafka/serializer/test/DA00HistogramSerializerTest.cpp [hunk 2]
```diff
@@ -7,12 +7,15 @@
 ///
 //===----------------------------------------------------------------------===//
 
-#include <common/kafka/serializer/FlatbufferTypes.h>
+#include "da00_dataarray_generated.h"
```
Comment (mortenjc) [inline src/common/kafka/serializer/test/DA00HistogramSerializerTest.cpp:10]: <>?
same here

File: src/common/kafka/serializer/test/DA00HistogramSerializerTest.cpp [hunk 3]
```diff
@@ -7,12 +7,15 @@
...
+#include "flatbuffers/flatbuffers.h"
```
Comment (mortenjc) [inline src/common/kafka/serializer/test/DA00HistogramSerializerTest.cpp:11]: and here

**Review by mortenjc — APPROVED** [review]

File: src/common/kafka/serializer/test/DA00HistogramSerializerTest.cpp (hunk 3 above)
Comment (tbukovics) [reply src/common/kafka/serializer/test/DA00HistogramSerializerTest.cpp:11]: thx, I will correct them. Sorry can be boring the same mistakes so many times.

**Pull request merged**